addons/account/models/account_report.py

```diff
@@ -589,19 +589,23 @@ def _expand_aggregations(self):
             domains = []
 
             for candidate_expr in to_expand:
-                labels_by_code = candidate_expr._get_aggregation_terms_details()
+                if candidate_expr.formula == 'sum_children':
```

Comment:
Do we need/want to restore the former tax report engine's behavior?
> no formula meant you had to sum the children

Reply:
We need a means to easily migrate custom-made tax reports, so sum_children will need to be put on the resulting expressions where there was no formula. No more ;)